backend/buildeR.DAL/Context/EntityConfigurations/TeamMemberConfig.cs

```diff
@@ -0,0 +1,23 @@
+﻿using buildeR.DAL.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+
+namespace buildeR.DAL.Context.EntityConfigurations
+{
+    public class TeamMemberConfig : IEntityTypeConfiguration<TeamMember>
+    {
+        public void Configure(EntityTypeBuilder<TeamMember> entity)
+        {
+            entity.HasOne(e => e.Group)
+                .WithMany(g => g.TeamMembers)
+                .HasForeignKey(e => e.GroupId)
+                .OnDelete(DeleteBehavior.NoAction);
+
+            entity.HasOne(e => e.User)
+                .WithMany(g => g.TeamMembers)
+                .HasForeignKey(e => e.UserId)
```

Comment:
Not needed, as it follows the convention for foreign keys: <Reference Navigation Property Name>Id

Reply:
oh, ok, I'll take it into account

Reply:
I still need to review this moment. Do I have to delete the ".HasForeignKey(e => e.UserId)" line, and the others where I wrote the same?

Reply:
Yep